### PR TITLE
Add leveled damage/healing to kineticists impulses

### PR DESCRIPTION
**Pull request by Zullock**
Attempt 2 after closing the last PR (https://github.com/foundryvtt/pf2e/pull/9129) on accident. Have already the changes suggested by @TikaelSol

### CODE REVIEW

File: packs/feats/aerial-boomerang.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
...
+            "value": "<p>A blade of shearing wind races away from you in a @Template[type:line|distance:60]. Each creature in the area takes @Damage[(floor((-1+@actor.level)/2 +2))d4[slashing]] damage with a basic Reflex save against your class DC. In the final square of the line, the boomerang whirls in place until the end of your next turn. Any creature that ends its turn in that square has to save against the boomerang.</p>\n<p>On your next turn, you can use a single action, which has the concentrate trait, to return the boomerang to you. It returns in a line from its square to your current location, with the same effect as the initial line, then the impulse ends. You must have line of effect to the boomerang and be within 60 feet of it.</p>\n<hr />\n<p><strong>Level (+2)</strong> The damage increases by 1d4.</p>"
```
Comment (stwlam) [inline packs/feats/aerial-boomerang.json:14]: prefer `@actor.level - 1` over the likes of `-1+@actor.level`

Comment (Zullock) [reply packs/feats/aerial-boomerang.json]: Made those changes.

File: packs/feats/ambush-bladderwort.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
...
...     "value": "<p>You plant the seed of a giant bladderwort in an unoccupied square of ground or water within 30 feet. It lasts for 10 minutes. Using this impulse again ends any previous one. If a creature enters its square, the seed erupts into a voluminous, translucent plant that seals tight around the triggering creature and fills with water. Unless the creature succeeds at a @Check[type:reflex|dc:resolve(@actor.system.attributes.classDC.value)] save against your class DC, it's @UUID[Compendium.pf2e.conditionitems.Item.Immobilized] within the bladderwort and must hold its breath or begin drowning. The @UUID[Compendium.pf2e.actionspf2e.Item.Escape] DC is also your class DC. The plant has AC 10 and 50 Hit Points.</p>\n<p>If a creature dies inside it, the plant shrinks down, converting itself and the remains into a watery fruit. A creature can eat this consumable to regain @Damage[((floor((@actor.level - 4)/4)+1)d8+(floor((@actor.level - 4)/4)*4+4))[healing]] HP, after which that creature is temporarily immune for 10 minutes. This fruit rots after 1 hour.</p>\n<hr />\n<p><strong>Level (+4)</strong> The bladderwort's HP increase by 25 and the fruit's healing increases by 1d8+4.</p>"
```
Comment (TikaelSol) [inline packs/feats/ambush-bladderwort.json:14]:
```suggestion
            "value": "<p>You plant the seed of a giant bladderwort in an unoccupied square of ground or water within 30 feet. It lasts for 10 minutes. Using this impulse again ends any previous one. If a creature enters its square, the seed erupts into a voluminous, translucent plant that seals tight around the triggering creature and fills with water. Unless the creature succeeds at a @Check[type:reflex|dc:resolve(@actor.system.attributes.classDC.value)] save against your class DC, it's @UUID[Compendium.pf2e.conditionitems.Item.Immobilized] within the bladderwort and must hold its breath or begin drowning. The @UUID[Compendium.pf2e.actionspf2e.Item.Escape] DC is also your class DC. The plant has AC 10 and 50 Hit Points.</p>\n<p>If a creature dies inside it, the plant shrinks down, converting itself and the remains into a watery fruit. A creature can eat this consumable to regain @Damage[((floor((@actor.level - 4)/4)+1)d8+(floor((@actor.level - 4)/4)*4+4))[healing]] HP, after which that creature is temporarily immune for 10 minutes. This fruit rots after 1 hour.</p>\n<hr />\n<p><strong>Level (+4)</strong> The bladderwort's HP increase by 25 and the fruit's healing increases by 1d8+4.</p>"
```

Inline healing isn't ready to be used for `@Damage` yet. We inadvertently caught it in some changes we made before but have since undone those.

Comment (Zullock) [reply packs/feats/ambush-bladderwort.json:14]: With the formulas is not being parsed correctly, should I put “leveled HP” or leave it as is?
image
image

Comment (TikaelSol) [reply packs/feats/ambush-bladderwort.json]: Put the baseline in the description and a leveled button with a label after that paragraph.

Comment (Zullock) [reply packs/feats/ambush-bladderwort.json]: Changed.

File: packs/feats/call-the-hurricane.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
...
+            "value": "<p>Massive waves spiral around you, with you as the eye of the hurricane. The waves appear in a @Template[type:emanation|distance:20], or a @Template[type:emanation|distance:30] if you're in a body of water. Each creature in the area takes @Damage[(floor((@actor.level - 8)/2)+6)d8[bludgeoning]] damage with a @Check[type:reflex|dc:resolve(@actor.system.attributes.classDC.value)|basic:true] save against your class DC. A creature that fails its save is battered by the waves and pushed 10 feet (or 20 feet on a critical failure).</p>\n<hr />\n<p><strong>Level (+2)</strong> The damage increases by 1d8.</p>"
```
Comment (TikaelSol) [inline packs/feats/call-the-hurricane.json:14]: `@Damage[(floor((@actor.level - 8)/2)+6)d8[bludgeoning]]` resolves to 2d8 when rendered in compendium (it assumes the actor level is 1). We can work around this by using the `max()` function. Two choices here are to just wrap the whole thing in that and feed it the minimum dice `@Damage[(max(6,floor((@actor.level - 8)/2)+6))d8[bludgeoning]]` but we can also use this directly on the actor level call to make sure the minimum number that resolves it 6d8 `@Damage[(floor((max(8,@actor.level)-8)/2)+6)d8[bludgeoning]]`

Unfortunately that's going to apply to most of these :(

Comment (TikaelSol) [reply packs/feats/call-the-hurricane.json:14]: I'm indifferent to whatever approach we take, the first is simpler the last is maybe cleaner looking but these formulas also tend to be hard to parse by hand anyways.

Comment (Zullock) [reply packs/feats/call-the-hurricane.json]: Added those changes, opted for the second one.
